hbase-client/src/test/java/org/apache/hadoop/hbase/client/TestMetricsConnection.java

```diff
@@ -146,111 +164,68 @@ public void testMetricsWithMultiConnections() throws IOException {
     impl.close();
   }
 
-  @Test
-  public void testMetricsConnectionScopeBlockingClient() throws IOException {
```

Comment:
why was this test removed?

Reply:
This is my fault, it has been rolled back.